client/components/data/query-reader-thumbnails/index.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { getThumbnailForIframe } from 'state/reader/thumbnails/selectors';
+import { requestThumbnail } from 'state/reader/thumbnails/actions';
+
+class QueryReaderThumbnails extends Component {
+	componentWillMount() {
```

Comment:
needs to handle `componentWillReceiveProps` too, in case the component is reused